crawl-ref/source/dat/des/branches/dis.des

```diff
@@ -823,31 +823,34 @@ MAP
 ............................7.........7.............................
 ENDMAP
 
-NAME:   dis_grunt
-MONS:   patrolling Dispater, Hell Sentinel, iron dragon / iron giant w:5
-MONS:   rust devil / iron imp w:20
-MONS:   iron troll, quicksilver dragon, cacodemon
-NSUBST: G = 4:8 / *:G
-TILE:   G = dngn_statue_iron_golem
-COLOUR: G = cyan
-KMONS:  8 = iron golem
-KMONS:  9 = war gargoyle
-ITEM:   plate armour good_item / crystal plate armour good_item w:3
-ITEM:   acquire weapon
-KMONS:  F = Hell Sentinel
-KITEM:  F = |
-KMONS:  E = Hell Sentinel
-KITEM:  E = $
+NAME:    dis_grunt
+MONS:    patrolling Dispater, Hell Sentinel, iron dragon / iron giant w:5
+MONS:    rust devil / iron imp w:20
+MONS:    iron troll, quicksilver dragon, cacodemon
+SHUFFLE: AP / BQ / CR / DS / ET
+SUBST:   A = 1, B = 2, C = 2, D = 7, E = 7
+SUBST:   P = O, Q = ., R = ., S = ., T = .
```

Comment:
Would this be clearer as

    SUBST: P = O, QRST = .

? I could see arguments either way: it makes it clear that only P matters, but on the other hand it hides the correspondence between A/P, B/Q, and so on. 

Similarly the previous line could be shortened if you think that's better.

Reply:
Yes I think that's simpler and more concise, I don't think the correspondence really matters.